generator/mavgen_java.py

```diff
@@ -84,7 +84,7 @@ def generate_CRC(directory, xml):
         data = data & 0xff; //cast because we want an unsigned type
         int tmp = data ^ (crcValue & 0xff);
         tmp ^= (tmp << 4) & 0xff;
-        crcValue = ((crcValue >> 8) & 0xff) ^ (tmp << 8) ^ (tmp << 3) ^ ((tmp >> 4) & 0xf);
+        crcValue = ((crcValue >> 8) & 0xff) ^ (tmp << 8) ^ (tmp << 3) ^ ((tmp >> 4) & 0xff);
```

Comment:
are you sure this is supposed to be 0xFF?

Reply:
I guess this is an error typo